### PR TITLE
Active property fixing and documentation

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Review by mkenttala — APPROVED** [review]

File: doc/sailfish-webview-webview.qdoc [hunk 1]
```diff
@@ -41,6 +39,18 @@
...
+/*!
+  \qmlproperty bool active
+  \brief This property is \c{true} if the WebView is currently active.
+
+  The active property follows application visibility and status of the WebViewPage unless explicitly set.
+  When the application or WebView is hidden (not visible) for the end user, active should be false.
+
+  Touching the content forces WebView to active state.
+
+  Normally application developer do not need to alter the default behaviour.
```
Comment (llewelld) [inline doc/sailfish-webview-webview.qdoc:51]: Just a really minor typo, but "application developers" (plural). Also, I'm not sure what the rule is, but I think elsewhere we use US-English (e.g. "initialize", "color"), in which case "behavior".

File: doc/sailfish-webview-webview.qdoc [hunk 2]
```diff
@@ -26,11 +26,9 @@
   ApplicationWindow {
       id: root
       initialPage: Component {
-          Page {
-              id: page
+          WebViewPage {
```
Comment (llewelld) [inline doc/sailfish-webview-webview.qdoc:29]: I've used this code as a base for other examples, so will aim to amend those in some other open PRs too.

Comment (rainemak) [reply doc/sailfish-webview-webview.qdoc:29]: Thanks

File: doc/sailfish-webview-webview.qdoc (hunk 1 above)
Comment (rainemak) [reply doc/sailfish-webview-webview.qdoc:51]: US-English. Will fix.

**Comment by rainemak** [comment]
Rebased and fixed @llewelld 's review findings.

**Comment by rainemak** [comment]
Review @okodron

**Review by okodron — APPROVED** [review]

**Review by llewelld — APPROVED** [review]

**Pull request merged**